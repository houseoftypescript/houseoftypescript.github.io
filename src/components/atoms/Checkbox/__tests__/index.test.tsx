import { render } from '@testing-library/react';
import { Checkbox } from '../index';

describe('Checkbox', () => {
  it('renders correctly', () => {
    const button = render(<Checkbox></Checkbox>);
    expect(button.container).toMatchSnapshot();
  });
});
