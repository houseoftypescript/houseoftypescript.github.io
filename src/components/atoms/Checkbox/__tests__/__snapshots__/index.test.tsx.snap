// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox renders correctly 1`] = `
<div>
  <label
    class="flex items-center gap-2"
    for=""
  >
    <input
      class="hidden"
      id=""
      name=""
      type="checkbox"
      value=""
    />
    <div
      class="w-5 h-5 rounded border border-2"
    />
    <span
      class="text-sm text-gray-500"
    />
  </label>
</div>
`;
